src/js/utilities/logIn.mjs

```diff
@@ -0,0 +1,42 @@
+import { setLoginFormDataCallback } from "./forms.mjs";
+
+function wrappedLogInUser() {
+  const API_BASE_URL = `https://api.noroff.dev/api/v1`;
```

Comment:
This would be better exported from a constants file, then imported wherever required.